README.md

```diff
@@ -66,6 +66,48 @@ build:mycluster-ubuntu16-04 --jobs=64
 EOF
 ```
 
+### Platform properties
+
+For clients to be able to use buildbarn, clients need to send actions with platform properties that
+match the platform properties that buildbarn workers register themselves with.
+
+For Bazel clients, platform properties can be set in the `.bazelrc` or as command line parameters:
```

Comment:
Though I guess you normally never set those, because bazel-toolchains sets those for you, right?

Reply:
Mm that's fair. Related to the comment about the `try a build section' being a bit lost, maybe this needs to be its own section, distinct from the remote-execution section?

Reply:
Moved it, let me know if that works for you.

Reply:
That place seems better. Thanks! Do make sure to explicitly mention that this isn’t needed for bazel-toolchain users.

Reply:
Yeh. I think that it would be helpful for bazel-toolchain users to document how the bazel platform referenced in `rbe_autoconfig`  relates to the platform defined in the runner configuration, so I've just expanded that out a bit.